sympy/solvers/simplex.py

```diff
@@ -889,84 +889,53 @@ def _handle_bounds(bounds):
     # introduce auxilliary variables as needed for univariate
     # inequalities
 
-    unbound = []
-    R = [0] * len(bounds)  # a (growing) row of zeros
-
-    def n():
-        return len(R) - 1
-
-    def Arow(inc=1):
-        R.extend([0] * inc)
-        return R[:]
+    def _make_list(length: int, index_value_pairs):
```

Comment:
Inside functions, the variable or symbol names are always hidden,
so we don't need underscore too much if it hurts readability of code

Reply:
I see. I’ll be more careful.